homeassistant/components/sensor/xbox_live.py

```diff
@@ -59,11 +69,16 @@ def __init__(self, hass, api, xuid):
         # get profile info
         profile = self._api.get_user_gamercard(self._xuid)
 
-        if profile.get('success', True) and profile.get('code', 0) != 28:
+        if profile.get('success', True) and profile.get('code', None) is None:
             self.success_init = True
             self._gamertag = profile.get('gamertag')
             self._picture = profile.get('gamerpicSmallSslImagePath')
         else:
+            _LOGGER.error("Can't get user profile %s. "
+                          + "Error Code: %s Description: %s",
```

Comment:
Remove `+`.

Reply:
Thank you for your feedback. I've implemented the requested change.